### PR TITLE
chore: add new colors and update docs

### PR DESCRIPTION
**Pull request by tassioFront**
## Infos

[Task](https://juntossomosmais.monday.com/boards/2413187753/pulses/2647640850)

#### What is being delivered?

Adding new neutral colors and updating documentation

#### What impacts?

- BU can start using new neutral colors
- DOCs using the right color variables

#### Reversal plan

change after merge

#### Evidences

Screen Shot 2022-05-25 at 18 16 15

## DoD checklist

- [ ] Related code finished
- [ ] Documentation updated (if exists)
- [ ] Unit tests written and passing
- [ ] Any configuration or build changes documented
- [ ] Project builds without errors
- [ ] Lint errors fixed
- [ ] Test a lib's generated build inside projects **!important**
- [ ] Update icon files (woff, woff2 and etc) from projects (if necessary).

### CODE REVIEW

**Review by schirrel — DISMISSED** [review]

**Review by mauriciomutte — APPROVED** [review]
🚀

**Review by igorwessel — APPROVED** [review]

File: packages/core/storybook/stories/2_Colors.stories.mdx [hunk 1]
```diff
@@ -10,56 +10,67 @@ All color palettes and their CSS Variables.
...
   />
   <ColorItem
-    title="Primary"
-    subtitle="Primary color, used in the most of cases"
+    title="Brand Primary"
     colors={[
-      'var(--primary-color)',
-      'var(--primary-color-lighten)',
-      'var(--primary-color-contrast)',
+      'var( --color-brand-primary-dark-1)',
```
Comment (munyrasirio) [inline packages/core/storybook/stories/2_Colors.stories.mdx:31]:
```suggestion
      'var(--color-brand-primary-dark-1)',
```

File: packages/core/storybook/stories/2_Colors.stories.mdx [hunk 2]
```diff
@@ -10,56 +10,67 @@ All color palettes and their CSS Variables.
...
     ]}
   />
   <ColorItem
     title="Secondary"
-    subtitle="Secondary color, used in some cases"
     colors={[
-      'var(--secondary-color)',
-      'var(--secondary-color-lighten)',
-      'var(--secondary-color-contrast)',
+      'var( --color-brand-secondary-dark-1)',
```
Comment (munyrasirio) [inline packages/core/storybook/stories/2_Colors.stories.mdx:40]:
```suggestion
      'var(--color-brand-secondary-dark-1)',
```

File: packages/core/storybook/stories/2_Colors.stories.mdx [hunk 3]
```diff
@@ -10,56 +10,67 @@ All color palettes and their CSS Variables.
...
     ]}
   />
   <ColorItem
     title="Success"
     subtitle="Success color, used for example in nice results"
     colors={[
-      'var(--success-color)',
-      'var(--success-color-lighten)',
-      'var(--success-color-contrast)',
+      'var( --color-contextual-success-dark-1)',
```
Comment (munyrasirio) [inline packages/core/storybook/stories/2_Colors.stories.mdx:50]:
```suggestion
      'var(--color-contextual-success-dark-1)',
```

File: packages/core/storybook/stories/2_Colors.stories.mdx [hunk 4]
```diff
@@ -10,56 +10,67 @@ All color palettes and their CSS Variables.
...
     ]}
   />
   <ColorItem
     title="Danger"
     subtitle="Danger color, used for example in errors"
     colors={[
-      'var(--danger-color)',
-      'var(--danger-color-lighten)',
-      'var(--danger-color-contrast)',
+      'var( --color-contextual-error-dark-1)',
```
Comment (munyrasirio) [inline packages/core/storybook/stories/2_Colors.stories.mdx:60]:
```suggestion
      'var(--color-contextual-error-dark-1)',
```

File: packages/core/storybook/stories/2_Colors.stories.mdx [hunk 5]
```diff
@@ -10,56 +10,67 @@ All color palettes and their CSS Variables.
...
+      'var(--color-contextual-error-regular)',
+      'var(--color-contextual-error-light-1)',
+      'var(--color-contextual-error-light-2)',
+    ]}
+  />
+  <ColorItem
+    title="Warning"
+    subtitle="Used for example in warnings"
+    colors={[
+      'var( --color-contextual-warning-dark-1)',
```
Comment (munyrasirio) [inline packages/core/storybook/stories/2_Colors.stories.mdx:70]:
```suggestion
      'var(--color-contextual-warning-dark-1)',
```

File: packages/core/storybook/stories/2_Colors.stories.mdx (hunk 1 above)
Comment (tassioFront) [reply packages/core/storybook/stories/2_Colors.stories.mdx]: Nice catch! https://github.com/juntossomosmais/venice/pull/483/commits/bdc33cafcd0e8c513fe685fff36b05fa204780a2

File: packages/core/storybook/stories/2_Colors.stories.mdx (hunk 2 above)
Comment (tassioFront) [reply packages/core/storybook/stories/2_Colors.stories.mdx]: Nice catch! https://github.com/juntossomosmais/venice/pull/483/commits/bdc33cafcd0e8c513fe685fff36b05fa204780a2

File: packages/core/storybook/stories/2_Colors.stories.mdx (hunk 4 above)
Comment (tassioFront) [reply packages/core/storybook/stories/2_Colors.stories.mdx]: Nice catch! https://github.com/juntossomosmais/venice/pull/483/commits/bdc33cafcd0e8c513fe685fff36b05fa204780a2

File: packages/core/storybook/stories/2_Colors.stories.mdx (hunk 5 above)
Comment (tassioFront) [reply packages/core/storybook/stories/2_Colors.stories.mdx]: Nice catch! https://github.com/juntossomosmais/venice/pull/483/commits/bdc33cafcd0e8c513fe685fff36b05fa204780a2

**Review by mauriciomutte — APPROVED** [review]

**Review by munyrasirio — APPROVED** [review]

**Review by schirrel — APPROVED** [review]

**Review by felipefialho — APPROVED** [review]
🚀 💘

**Pull request merged**